src/meta.js

```diff
@@ -21,12 +22,17 @@ export function getSDKMeta(): string {
   const url = getScriptUrl();
 
   const scriptAttrs = getSDKAttributes();
+  const handler = () => {};
+  const shopperInsights = getSessionState(handler);
```

Comment:
```suggestion
  const shopperInsightsUsed = getSessionState(state => state.shopperInsights.getRecommendedPaymentMethodsUsed)
```

https://github.com/paypal/paypal-checkout-components/pull/2286/files#diff-6d1caeaa7a4f194561101db7781e6ff562dd9a95c9cb07852bc69ce6443af15eR18